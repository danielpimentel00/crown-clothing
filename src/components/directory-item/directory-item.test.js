import { shallow } from "enzyme";
import DirectoryItem from "./directory-item.component";
import { BrowserRouter as Router } from "react-router-dom";

describe("Directory Item Component", () => {
  it("should match snapshot", () => {
    const mockProps = {
      title: "",
      imageUrl: "",
      route: "../",
    };
    const wrapper = shallow(
      <Router>
        <DirectoryItem category={mockProps} />
      </Router>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
