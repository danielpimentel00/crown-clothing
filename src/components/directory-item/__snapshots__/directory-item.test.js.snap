// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Directory Item Component should match snapshot 1`] = `ShallowWrapper {}`;
